setup.py

```diff
@@ -21,8 +21,12 @@
     long_description_content_type="text/markdown",
     name="pip-api",
     packages=find_packages(),
+    package_data={
+        "pip_api": ["py.typed"],
+    },
     python_requires=">=3.6",
     url="http://github.com/di/pip-api",
     summary="An unofficial, importable pip API",
     version="0.0.20",
+    zip_safe=False,
```

Comment:
N.B.: I'm not 100% sure if this is necessary, but it's what `mypy`'s docs recommend when including `py.typed`.

Reply:
The `zip_safe` thing shouldn't be necessary here: https://github.com/python/mypy/issues/8802